src/renderer/viz/expressions/ramp.js

```diff
@@ -145,7 +144,7 @@ export default class Ramp extends BaseExpression {
     }
 
     _getColorValue (texturePixels, m) {
-        const index = Math.round(m * MAX_BYTE_VALUE);
+        const index = _calcColorValueIndex(m);
```

Comment:
Could you add a unit test to cover this one?

Reply:
sure! ✨